app/static/css/_form.scss

```diff
@@ -98,3 +98,58 @@
 .date-picker__group {
     flex: 0 1 auto;
 }
+
+
+.loading{
+    width:00px; 
```

Comment:
I think the background on this might work better as `$soft-white` -can you give that a try please?